ticTacToe.test.ts

```diff
@@ -8,112 +8,176 @@ describe("tic tac toe game:", () => {
     ticTacToe = new TicTacToe();
   });
 
-  test("it should have empty board", () => {
-    expect(ticTacToe.getBoardGame()).toEqual([
+  it("it should have empty board", () => {
+    expect(ticTacToe.gameBoard).toEqual([
       ["", "", ""],
       ["", "", ""],
       ["", "", ""],
     ]);
   });
 
-  test("player can make a move", () => {
-    const player = "X";
-    ticTacToe.makeMove(0, 0);
-    expect(ticTacToe.getBoardGame()[0][0]).toEqual(player);
-  });
+  describe("providing custom game board to constrcutor", () => {
+    it("should create empty game board by providing to consturctor", () => {
```

Comment:
this test doesn't check anything because the empty board is the default (I mean, that even if passing the board to the constructor won't work, it will still pass as it will load the default). the right test should try to init the game with a board different from the default one.